templates/python-poetry/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) bakdata
```

Comment:
In [the Java project](https://github.com/bakdata/bakdata-project-templates/blob/master/templates/java-gradle/LICENSE), there's also a copyright year here which seems to get templated somehow: `Copyright (c) {{year}} bakdata`

Reply:
hm, I am not a copyright lawyer but I've seen many open source projects abandon the year in their copyright notice/license, so I would suggest we do the same

Reply:
I think the license itself is still valid even without the year in it. Therefore I would skip it